frontend-react/src/hooks/UseSenderResource.ts

```diff
@@ -12,7 +12,9 @@ export type UseSenderResourceHookResult = {
     isInitialLoading: boolean;
 };
 
-export const useSenderResource = (): UseSenderResourceHookResult => {
+export const useSenderResource = (
+    initialData?: any
+): UseSenderResourceHookResult => {
     const { authorizedFetch, rsUseQuery } = useAuthorizedFetch<RSSender>();
```

Comment:
Just a thought, instead of calling `rsUseQuery` could we call `wrapUseQuery` and pass true to check if initialized?
Then you wouldn't need the initialData prop.